joss-paper/paper.md

```diff
@@ -0,0 +1,160 @@
+and for CW-like long-duration transient signals e.g. from _after_ a pulsar glitch.
+Specialized versions of both grid-based and MCMC-based search classes
+are provided for these scenarios.
+Both fully-coherent and semi-coherent searches
+(where the data is split into several segments for efficiency)
+are covered,
+and an extension to the $\mathcal{F}$-statistic
+that is more robust against single-detector noise artifacts
+[@Keitel:2013wga]
+is also supported.
+While `PyFstat`'s grid-based searches do not compete with the sophisticated
+grid setups and semi-coherent algorithms implemented in various `LALSuite` programs,
+its main scientific use cases so far are for the MCMC exploration
+of interesting parameter-space regions
+and for the long-duration transient case.
+
+`PyFstat` was first introduced in @Ashton:2018ure which remains the main reference
+for the MCMC-based analysis implemented in the package.
+The extension to transient signals, which uses `PyCUDA` [@Kloeckner:2012pyc] for speedup,
+is discussed in detail in @Keitel:2018pxz,
+and the glitch-robust search approaches in @Ashton:2018qth.
+
+Additional helper classes, utility functions and internals are included for
+handling the common Short Fourier Transform (SFT) data format for LIGO data,
+simulating artificial data with noise and signals in them,
+and plotting results and diagnostics.
+Most of the underlying `LALSuite` functionality is accessed through SWIG wrappings [@Wette:2020air]
+though for some parts, such as the SFT handling,
+we still (as of the writing of this paper) call stand-alone `lalapps` executables.
+Completing the backend migration to pure SWIG usage is planned for the future.
+
+The source of `PyFstat` is hosted on [GitHub](https://github.com/PyFstat/PyFstat/).
+The repository also contains an automated test suite
+and a set of introductory example scripts.
+Issues with the software can be submitted through GitHub
+and pull requests are always welcome.
+Documentation in html and pdf formats is available from [readthedocs.org](https://readthedocs.org/projects/pyfstat/)
+and installation instructions can be found there
+or in the [README](https://github.com/PyFstat/PyFstat/blob/master/README.md) file.
+
+
+# Statement of need
+
+The sensitivity of searches for CWs and long-duration transient GWs
+is generally limited by computational resources,
+as the required number of matched-filter templates increases steeply
+for long observation times and wide parameter spaces.
+The C-based `LALSuite` library [@lalsuite] contains many sophisticated search methods
+with a long development history and high level of optimization,
+but is not very accessible for researchers new to the field or for students;
+nor is it very convenient for rapid development and integration with modern technologies
+like GPUs or machine learning.
+Hence, `PyFstat` serves a dual function of
+(i) making `LALSuite` CW functionality more easily accessible through a `Python` interface,
```

Comment:
`Python` got broken (Pyth-on) in my version of the PDF, not sure there is a way to avoid that from happening?

Reply:
Thanks for spotting that. Since this goes through a markdown->tex conversion, I have no idea how to properly deal with it, and settled for the good old "delete a small superfluous word earlier in the paragraph" solution. ;)